Doc/library/inspect.rst

```diff
@@ -153,6 +153,19 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
 |                 | f_trace           | tracing function for this |
 |                 |                   | frame, or ``None``        |
 +-----------------+-------------------+---------------------------+
+|                 | f_trace_lines     | indicate whether a        |
+|                 |                   | tracing event is          |
+|                 |                   | triggered for each source |
+|                 |                   | source line               |
++-----------------+-------------------+---------------------------+
+|                 | f_trace_opcodes   | indicate whether          |
+|                 |                   | per-opcode events are     |
+|                 |                   | requested                 |
++-----------------+-------------------+---------------------------+
+|                 | clear             | used to clear all         |
```

Comment:
Would it make sense to add parenthesis to methods? Such as: `clear()`.

Reply:
I somehow lost this during the previous stage of the review...
Yes, that definitely makes things more obvious. Additionally, I will add parentheses to the docstrings.